sys/linux/dev_kvm.txt

```diff
@@ -261,10 +268,20 @@ syzos_api_msr {
 	arg_value	int64
 }
 
+kvm_smc_id = 0x80000000, 0x40000000, 0x1000000, 0x2000000, 0x3000000, 0x4000000, 0x30000000, 0x31000000, 0x32000000, 0x3f000000, 0xffff, 0x0, 0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8, 0x10, 0x20, 0x40, 0x80, 0x100, 0x200, 0x400, 0x800, 0x1000, 0x2000, 0x4000, 0x8000, 0x10000
```

Comment:
Nit: it would help to have a comment here re. what are these and where they are from.

Reply:
Done